docs/articles/documentation/test-api/actions/upload.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+title: Upload
+permalink: /documentation/test-api/actions/upload.html
+---
+# Upload
+
+Use the following actions to work with file upload input elements.
+
+* [Populate File Upload Input](#populate-file-upload-input)
+* [Clear File Upload Input](#clear-file-upload-input)
+
+## Populate File Upload Input
+
+Populates the specified file upload input with file paths.
+
+```text
+setFilesToUpload( selector, filePath )
+setFilesToUpload( selector, [filePath, ...] )
+```
+
+Parameter  | Type                                              | Description
+---------- | ------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------
+`selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the input field to which file paths are written. See [Selecting Target Elements](index.md#selecting-target-elements).  
+`filePath` | String                                            | The path to the uploaded file. Relative paths resolve from the folder with the test file.
```

Comment:
Hmmm... When I asked this question

> Or do you think that src([source, ...]) is non-standard and confused with optional arguments?

before it was okay
https://github.com/DevExpress/testcafe/pull/298#discussion_r50427565